db/migrate/20160614121943_move_public_to_visibility_on_namespace.rb

```diff
@@ -0,0 +1,13 @@
+class MovePublicToVisibilityOnNamespace < ActiveRecord::Migration
+  def up
+    Namespace.all.each do |namespace|
+      namespace.visibility = namespace.public? ? :visibility_public : :visibility_private
```

Comment:
What about existing activities ? I think that we should offer a way to migrate from those as well.